package.json

```diff
@@ -168,7 +168,7 @@
     "react-native-screens": "3.34.0",
     "react-native-share": "11.0.2",
     "react-native-sound": "^0.11.2",
-    "react-native-svg": "15.6.0",
+    "react-native-svg": "15.9.0",
```

Comment:
I had to bump react-native-svg, because styling <Svg /> components was broken on android in previous versions.